islandora_transforms/datastream_info_to_solr.xslt

```diff
@@ -9,9 +9,6 @@
     <field name="fedora_datastreams_ms">
       <xsl:value-of select="@ID"/>
     </field>
-    <field name="fedora_datastreams_mimetypes_ms">
```

Comment:
This seems to be a regression of https://github.com/discoverygarden/islandora_transforms/commit/5f374290c5836063f7a062e2f4291136a176b195